API/Repositories/Interfaces/IMatchesRepository.cs

```diff
@@ -15,6 +15,7 @@ public interface IMatchesRepository : IRepository<Match>
 	Task<Match?> GetFirstMatchNeedingAutoCheckAsync();
 	Task<IList<Match>> GetNeedApiProcessingAsync();
 	Task<IEnumerable<Match>> GetByMatchIdsAsync(IEnumerable<long> matchIds);
+	Task<Match> UpdateVerificationStatus(int id, int? verificationStatus);
```

Comment:
`verificationStatus` should be an `int` on update (not `int?`). I'm honestly not sure if there is ever a case where `verificationStatus` is null, will have to investigate separately.

Reply:
Gotcha, I think changing the method to accept an int and using the null forgiving operator when passing it in should probably be fine for now. Like this: `(int)match.VerificationStatus!`.

Reply:
Null ignore should be fine but doesn't feel optimal.

Reply:
I'll check db and see if this field should be nullable.